app/assets/stylesheets/nucore.scss

```diff
@@ -354,3 +354,12 @@ textarea.wide {
     display: inherit;
   }
 }
+
+/* Bootstrap adds an element with this class when you expand a dropdown and your
+ * browser supports the ontouchstart event. It claims to be necessary "if mobile
+ * we we use a backdrop because click events don't delegate". Instead, this
+ * is taking the tap and preventing the actual link to receive the click.
+ */
+.dropdown-backdrop {
+  display: none;
+}
```

Comment:
I found this by using the web inspector with the simulator. This explains why they do it, so maybe this is not right solution: https://v4-alpha.getbootstrap.com/components/dropdowns/#usage